src/remote/run_jobs.jl

```diff
@@ -0,0 +1,198 @@
+using Snowflake
+using JSON
+using HTTP
+
+commonpath="benchmarking/data"
+
+function serialize_circuit(circuit::QuantumCircuit,repetitions::Integer;indentation::Integer=4)
```

Comment:
You should never indent JSON that is sent over the network. If you want to be able to debug this perhaps return `circuit_description` instead and call `JSON.json(...)` when making the request in `submit_circuit`.